projects/pgai/pgai/cli.py

```diff
@@ -117,13 +117,22 @@ def convert(self, value, param, ctx) -> int:  # type: ignore
             )
 
 
+def get_log_level(level: str) -> int:
+    level_upper = level.upper()
+    # We are targeting python 3.10 that's why we need to use getLevelName which
+    # is deprecated, but still there for backwards compatibility.
+    level_name = logging.getLevelName(level_upper)  # type: ignore
+    if level_upper != "INFO" and isinstance(level_name, int):
+        return level_name
+    return logging.getLevelName("INFO")  # type: ignore
+
+
 @click.command(name="worker")
 @click.version_option(version=__version__)
 @click.option(
     "-d",
     "--db-url",
     type=click.STRING,
-    envvar="VECTORIZER_DB_URL",
```

Comment:
The docs need to be updated to reflect the change in env var name.
https://github.com/timescale/pgai/blob/vectorizer/docs/self-hosting-vectorizer.md?plain=1#L42

Reply:
That's part of @billy-the-fish PR

Reply:
https://github.com/timescale/pgai/pull/158#discussion_r1813163360

Reply:
ahh. ok